index.js

```diff
@@ -0,0 +1,90 @@
+$(document).ready(function() {
+  $('#allTrips').on('click', function() {
+    let $allTrips = $(this);
```

Comment:
use `const` instead of `let` since you don't reassign `$allTrips`, but good work naming it with a dollar sign.  That makes it clear this is a jQuery reference.